integration_tests/seeds/first_in_between/output/output__first_in_between_1.csv

```diff
@@ -1,3 +1,3 @@
-activity_id,entity_uuid,ts,activity,anonymous_entity_uuid,feature_json,revenue_impact,link,activity_occurrence,activity_repeated_at,first_in_between_bought_something_feature_json,first_in_between_bought_something_ts
-3,1,2022-01-02 22:10:11,signed up,,"{""type"": 1}",0,,1,,"{""type"": 1}",2022-01-05 22:10:11
```

Comment:
Note how the larger `revenue_impact` of 100 is correctly selected, because it is first in the `input__first_in_between` seed. Before the min would select 99, the later impact, incorrectly, because the aggregation was operating on the `revenue_impact` column directly, instead of the `ts` prepended column.